city_scrapers/spiders/losca_City_Planning.py

```diff
@@ -0,0 +1,101 @@
+import re
+
+from city_scrapers_core.constants import BOARD, COMMISSION, NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from dateutil.parser import parse
+
+
+class LoscaCityPlanningSpider(CityScrapersSpider):
+    name = "losca_City_Planning"
+    agency = "Los Angeles City Planning"
+    timezone = "America/Los_Angeles"
+    start_year = 2024
+    start_urls = [
+        f"https://planning.lacity.gov/dcpapi/meetings/api/all/commissions/{start_year}",
+        f"https://planning.lacity.gov/dcpapi/meetings/api/all/boards/{start_year}",
+        f"https://planning.lacity.gov/dcpapi/meetings/api/all/hearings/{start_year}",
+    ]
```

Comment:
When I ran the test command, I realized it got less meetings than what are shown on the [source website](https://planning.lacity.gov/about/commissions-boards-hearings), eg. last meeting of the year 2024 [Area Planning Commission - 12/26/2024](https://planning.lacity.gov/dcpapi2/meetings/document/77972) is missing. It turned out that the website is using API v2 endpoints, while you're using API v1. 
So please update `start_urls` to API v2 endpoints:
```python
start_urls = [
    f"https://planning.lacity.gov/dcpapi2/meetings/api/all/commissions/{start_year}",
    f"https://planning.lacity.gov/dcpapi2/meetings/api/all/boards/{start_year}",
    f"https://planning.lacity.gov/dcpapi2/meetings/api/all/hearings/{start_year}",
]
```

Please see my test outputs for each `start_url` API v2 endpoint:
[test_output_all_hearings_v2.csv](https://github.com/user-attachments/files/18243454/test_output_all_hearings_v2.csv)
[test_output_boards_v2.csv](https://github.com/user-attachments/files/18243455/test_output_boards_v2.csv)
[test_output_commissions_v2.csv](https://github.com/user-attachments/files/18243457/test_output_commissions_v2.csv)

❗ One thing I noticed with the API v2 endpoints is that they seem to have rate limit. To produce those 3 test outputs files, I couldn't run the test commands continuously. Sometimes I got zero-byte output file, so I had to wait for about 1-2 minutes to rerun the command, and it was successful eventually. So when you update the API v2 endpoints, please make sure to take care of this issue.